### PR TITLE
1304 newcity landing page

### LINKED ISSUE #1304: Need to update/remake the landing page for Project Sidewalk
When we deploy to multiple cities, we will need to update the landing page ([http://projectsidewalk.io](http://projectsidewalk.io)) that allows users to choose which cities they want to audit. This should redirect to other cities (like sidewalk.cs.washington.edu/seattle, sidewalk.cs.washington.edu/dc, etc...). Maybe a feature that allows users to select the cities they are closest to would be nice as well.

### PR DESCRIPTION
**Pull request by misaugstad**
Partially resolves #1304 

Adds links to the audit pages for other cities to the landing page.

Before
landing-page-no-links

After
landing-page-with-links

Pretty easy change, so I won't ask anyone to test. Unless you specifically want to take a look @manaswis to make sure this meets your expectations.

### CODE REVIEW

**Comment by manaswis** [comment]
Awesome! Checking it now!

**Comment by manaswis** [comment]
image

For me, the second link loads as Seattle, WA

**Comment by misaugstad** [comment]
well what city are you using..? If you are using DC then that is expected.

**Comment by manaswis** [comment]
Do I need to change the config?

image

So the button loads Washington DC but the button says Seattle. So I am guessing there is some config issue.

**Comment by misaugstad** [comment]
Oh right you also hard-coded the "Start Exploring Seattle" -- I need to update that.

**Comment by misaugstad** [comment]
I forgot that didn't just say "Start Exploring" still

File: app/views/index.scala.html [hunk 1]
```diff
@@ -33,12 +33,10 @@
                     <br>
                     <a class="bodyStartBtn" href="@routes.AuditController.audit()">Start Exploring Seattle</a>
```
Comment (manaswis) [inline app/views/index.scala.html]: Oh this is the culprit. The city name is hard-coded.

Comment (misaugstad) [reply app/views/index.scala.html]: yep

**Comment by misaugstad** [comment]
pull changes and try again

**Comment by misaugstad** [comment]
and if you want to switch to a different city, change you SIDEWALK_CITY_ID environment variable to either "newberg-or" or "seattle-wa"

**Comment by manaswis** [comment]
Wonderful! Works perfectly! Thanks @misaugstad

**Pull request merged**